src/Agent.Worker/Release/ReleaseJobExtension.cs

```diff
@@ -212,9 +205,10 @@ public ReleaseJobExtension()
                 await retryExecutor.ExecuteAsync(
                     async () =>
                         {
-                            //TODO:SetAttributesToNormal
```

Comment:
I don't remember why we had this TODO. seems fine to remove, but would be good to check once with Omesh if he had anything specific...